__tests__/data/salesforce_proxy_output.json

```diff
@@ -0,0 +1,125 @@
+[
+  {
+    "output": {
+      "status": 204,
```

Comment:
The `204` http-status-code can cause problems in `rudder-server`(go's http client). So, basically, we can conclude a `204` from Salesforce is equivalent to `200`

Reply:
yes, updated, for all successful responses i.e. <400, we will convert to 200 as status.